### DISCUSSION TITLE
Analyse des Génomes 2024 -- Checkpoint 3

### DISCUSSION THREAD

**Discussion by drosofff** [ARTbio/AnalyseGenome#42] [General]
### Checking the availability of required input datasets in your Galaxy server

Please copy below (one comment by student) a screenshot of your `History Multiview`, as in the first comment of the discussion

**Comment by drosofff** [comment]
<img width="797" alt="Capture d’écran 2024-11-18 à 10 54 40" src="https://github.com/user-attachments/assets/5225d367-e896-46dc-8cd6-0e6b0f8678f3">

**Comment by Baptiste-Dm** [comment]
Bonsoir,

Voici mes historiques Galaxy:
![{40CC3AD1-9274-4D76-9D92-52FE8F0154F9}](https://github.com/user-attachments/assets/2cd7aa61-2ee4-45c5-8d81-69fb8acde698)

**Reply by drosofff** [reply]
Je suppose que vous avez suivi les instructions de @slecrom pour un chargement des données plus restreint, @Baptiste-Dm .

Quoique il en soit, il me parait un peu risqué d'avoir des nomenclatures très similaires entre sRNAseq et RNAseq data. Attention à ne pas vous tromper plus tard...

**Reply by slecrom** [reply]
@drosofff tu as raison c'est risqué, il faudra le dire à Antoine et Laure :wink: Ce sont les numéros de référence qui changent. 
Merci @Baptiste-Dm pour le partage de la copie d'écran. Vous êtes prêt pour la semaine prochaine.

**Comment by Juleslr22** [comment]
Bonjour, 
Voici mon historique galaxy :

![Capture d'écran 2024-11-26 114445](https://github.com/user-attachments/assets/825dfa82-0a81-4171-9bdd-6160f93de221)

**Reply by drosofff** [reply]
C'est normal que les collections de l'histoire `Small RNA sequencing data` et de l'histoire `RNA sequence datasets` aient les mêmes noms (WT et GLKD) ? Si oui, attention au tapis @Juleslr22 ...

**Reply by Juleslr22** [reply]
Oui c'est normal dans le sens où c'est ce que je voulais faire mais vous avez raison je vais les renommer pour mieux les différencier.

**Reply by slecrom** [reply]
Bonjour @Juleslr22 
Vous avez chargé toutes les données. Mais pour le TP vous n'allez travailler qu'avec 2 échantillons comme indiqué dans les encadrés orange dans le tutoriel. Je vous propose d'extraire de vos collections les échantillons sur lesquels vous allez travailler. Vous pouvez suivre les instructions disponibles via le lien ci-dessous pour savoir comment faire.
https://slecrom.github.io/AG2024/annexes/#extraire-des-elements-dune-collection
Bonne journée

**Comment by Maille0** [comment]
Grégoire Blavier

![Checkpoin3_GB](https://github.com/user-attachments/assets/a1bddfc0-5a5a-4f8e-943a-9d7afd9cfcfd)

**Reply by slecrom** [reply]
Bonjour @Maille0 
Vous avez chargé toutes les données. Mais pour le TP vous n'allez travailler qu'avec 2 échantillons comme indiqué dans les encadrés orange dans le tutoriel. Je vous propose d'extraire de vos collections les échantillons sur lesquels vous allez travailler. Vous pouvez suivre les instructions disponibles via le lien ci-dessous pour savoir comment faire.
https://slecrom.github.io/AG2024/annexes/#extraire-des-elements-dune-collection
Bonne journée

**Comment by Emmabenbakir** [comment]
<img width="816" alt="Capture d’écran 2024-11-26 à 18 29 01" src="https://github.com/user-attachments/assets/cd7638bb-f94d-4ab6-815d-3eabeb918927">

**Reply by slecrom** [reply]
Bonjour @Emmabenbakir 
Vous avez chargé toutes les données des ARN. Mais pour le TP vous n'allez travailler qu'avec 2 échantillons comme indiqué dans les encadrés orange dans le tutoriel. Comme les échantillons individuels sont visibles dans votre histoire ce n'est pas grave.
Bonne journée

**Reply by slecrom** [reply]
@Emmabenbakir je ne vois pas l'échantillon "Test Mapping" dans l'histoire des ARN, est-ce que vous l'avez téléchargée ?

**Reply by Emmabenbakir** [reply]
Oui, effectivement dans RNA sequence data, il y a bien Test Mapping, ça ne se voit pas sur le premier screen shot. 
![image](https://github.com/user-attachments/assets/6029d60d-2811-433e-9ed6-1d99017a2d2c)

**Reply by slecrom** [reply]
Merci @Emmabenbakir nous verrons lundi pour finaliser l'utilisation des fichiers

**Comment by YoannGS** [comment]
Bonjour, voici mes historiques Galaxy :
![Screenshot 2024-11-26 at 20-00-03 Galaxy 🧬🔬🚀](https://github.com/user-attachments/assets/5395be63-b90d-4d30-961f-cf1b936e6eb0)

**Reply by slecrom** [reply]
Bonjour @YoannGS 
Vous avez chargé toutes les données. Mais pour le TP vous n'allez travailler qu'avec 2 échantillons comme indiqué dans les encadrés orange dans le tutoriel. Pour les ARN, je vous propose d'extraire de vos collections les échantillons sur lesquels vous allez travailler. Vous pouvez suivre les instructions disponibles via le lien ci-dessous pour savoir comment faire.
https://slecrom.github.io/AG2024/annexes/#extraire-des-elements-dune-collection
Pour les petits ARN comme vous n'avez pas fait de collections, laissez votre historique tel que.
Bonne journée

**Reply by YoannGS** [reply]
Bonjour @slecrom,
est-ce bon comme ça ?
NB : je viens de remarquer qu'il manque Test-mapping, je viens de le rajouter, 
![image](https://github.com/user-attachments/assets/199bdca6-e9a5-440f-893e-1414a08daa21)

**Comment by MathildeQuib** [comment]
![image](https://github.com/user-attachments/assets/ac2cdcf0-0075-42ca-9cd9-8b055f885b57)

**Reply by slecrom** [reply]
Bonjour @MathildeQuib 
Pouvez-vous prendre une copie d'écran plus large pour que je puisse voir le contenu complet de vos historiques ?
Merci

**Reply by MathildeQuib** [reply]
![image](https://github.com/user-attachments/assets/cdc2cb76-3ab2-4863-b2b0-a336999e0c6b)

**Reply by slecrom** [reply]
Merci @MathildeQuib vous êtes prête pour lundi.

**Comment by oceanewthr** [comment]
![image](https://github.com/user-attachments/assets/2d5dfcd6-49bd-4063-87c5-19616e34d1f9)

**Reply by slecrom** [reply]
Bonjour @oceanewthr 
Vous avez chargé toutes les données. Mais pour le TP vous n'allez travailler qu'avec 2 échantillons comme indiqué dans les encadrés orange dans le tutoriel. Je vous propose d'extraire de vos collections les échantillons sur lesquels vous allez travailler. Vous pouvez suivre les instructions disponibles via le lien ci-dessous pour savoir comment faire.
https://slecrom.github.io/AG2024/annexes/#extraire-des-elements-dune-collection
Bonne journée

**Comment by sarah-grn** [comment]
![Checkpoint 3](https://github.com/user-attachments/assets/e5d52c76-78ff-4b98-88a9-bd560606e1e1)

**Reply by slecrom** [reply]
Bonjour @sarah-grn 
Il vous manque l'échantillon "Test-mapping" dans l'histoire RNA. Pouvez-vous l'uploader sur votre serveur ?
Bonne journée.

**Comment by LoannPtr** [comment]
Bonsoir, voici mes données:

![image](https://github.com/user-attachments/assets/c0460e51-bd93-43f3-af6d-8b4630f81f9e)

**Reply by slecrom** [reply]
Bonjour @LoannPtr 
Merci pour le partage de votre copie d'écran. L'extension de fichier de la séquence PLacZ est étrange. Elle devrait être au format Fasta. Pouvez-vous la télécharger de nouveau sur votre serveur Galaxy à partir du lien du tutoriel ?
Bonne journée,
Stéphane

**Reply by LoannPtr** [reply]
Bonjour, normalement c'est réglé !

**Comment by fabouraima** [comment]
<img width="1455" alt="Capture d’écran 2024-11-27 à 18 04 56" src="https://github.com/user-attachments/assets/e5148f22-7ca0-483e-8851-5edee321481a">, 

Voici mes données

**Reply by drosofff** [reply]
Dites-donc, @fabouraima , c'est moche ça non 😄 ?
Vous voyez bien que vous avez changé quelque chose dans la manière d'uploader les données entre l'histoire `Références` et les histoires des RNAseq et sRNAseq...

**Reply by slecrom** [reply]
Bonjour @fabouraima 
Le problème vient je pense d'une erreur dans l'affectation des noms des colonnes au moment de l'import. Vous avez deux options. Soit effacer les données et les recharger. Soit les renommer.
Pouvez-vous recharger votre copie d'écran, en ne conservant que la vue des 3 histoires ? Je n'ai pas besoin de voir le reste de la page.
Bonne journée.

**Comment by ldebarbarin** [comment]
Bonjour, Voici mon screenshot

![image](https://github.com/user-attachments/assets/effa814f-9a3f-40e3-94eb-7235728f3e3a)

**Reply by slecrom** [reply]
Bonjour @ldebarbarin 
Parfait. Vous êtes prête pour lundi.
Bonne journée.

**Comment by Enzobcrl** [comment]
Bonjour, voici mon Screenshot de datasets
<img width="799" alt="Screenshot 2024-11-28 at 00 55 12" src="https://github.com/user-attachments/assets/13fb9154-d97e-44e5-b30f-59a1d562ca05">

**Reply by slecrom** [reply]
Bonjour @Enzobcrl 
Merci pour la capture d'écran. C'est bon pour lundi.
Bonne journée.

**Comment by ldebarbarin** [comment]
Bonjour,
Je me permets d'envoyer un message car j'ai réalisé une erreur lors de la création de ma VM. Je n'avais pas remarqué les différentes localisations d'hébergements assignées (Belgique pour moi) et ai donc renseigné zurich comme sur l'exemple. 
Dois-je supprimer ma VM et en créer une nouvelle ou est ce bon tout de même ?
Merci

**Reply by drosofff** [reply]
Et voilà, tout le tp est foutu ! @ldebarbarin

**Reply by drosofff** [reply]
Je plaisante... Ca ne pose pas de problème, gardez votre VM !

**Comment by mhully** [comment]
Bonjour, voici les datasets téléchargés :
![Capture d'écran 2024-11-27 172214](https://github.com/user-attachments/assets/7109fe69-7778-435c-94ba-7f77b85f14fb)

**Reply by slecrom** [reply]
Merci @mhully vous êtes prêt pour lundi.

**Comment by sarah-grn** [comment]
Bonjour @slecrom,
Voici la capture de mes datasets suite a l'upload de l'échantillon "Test-mapping" sur mon serveur. 
Bonne journée.

![Checkpoint3-SG](https://github.com/user-attachments/assets/f3fb8023-10a3-4be9-b701-035314600e7e)

**Reply by slecrom** [reply]
Merci @sarah-grn vous êtes prête pour lundi.

**Comment by fabouraima** [comment]
Bonjour, 

Est ce que c'est tout bon  maintenant? @drosofff ^^
<img width="1174" alt="Capture d’écran 2024-11-28 à 13 01 05" src="https://github.com/user-attachments/assets/0f3cd494-cebf-49a2-bdde-8d573cb0989d">

**Reply by drosofff** [reply]
Oh c'est ravissant ❤️ 💟 😍 @fabouraima !

**Comment by MaeDrai** [comment]
Bonjour, 
Voici la capture de mon History Multiview : 

<img width="550" alt="History_multiview_MD" src="https://github.com/user-attachments/assets/f17aa3ad-0c2b-4aeb-b07b-6fccc1bd7fe1">

A lundi !

**Reply by slecrom** [reply]
Merci @MaeDrai tout est bon pour moi, vous êtes prêt pour lundi.

**Comment by alleongaelle** [comment]
Bonsoir, 
Voici les historiques de mon serveur Galaxy: 

![image](https://github.com/user-attachments/assets/7664fd92-3ac8-4772-9669-c80035de8835)

Bonne soirée et à lundi !

**Reply by slecrom** [reply]
Bonjour @alleongaelle 
Merci pour le partage d'écran. Tout est bon. Vous êtes prête pour lundi.
Bonne journée,

**Comment by sbensous** [comment]
Bonjour, Voici l'historique de mon serveur : 
![Capture d’écran 2024-11-28 215218](https://github.com/user-attachments/assets/4f35aba7-fd14-493e-b77c-f09b1bc830cc)

**Reply by sbensous** [reply]
J'ai vue qu'après votre message pour la décompression des fichiers mais j'ai décidé de garder les fichiers compressés en attendant pour m'assurer que le jour du TP tout est bon !

**Reply by slecrom** [reply]
Bonjour @sbensous 
Ce n'est pas grave pour la décompression. On reverra tout cela lundi.
Bonne journée,

**Comment by n-lacombe** [comment]
Bonsoir, voici mes historiques Galaxy :

![image](https://github.com/user-attachments/assets/ff1bdd30-e83e-4709-8ddd-aa1fc5617bd7)

**Reply by slecrom** [reply]
Bonjour @n-lacombe 
Merci pour le partage d'écran. Tout est bon. Vous êtes prêt pour lundi.
Bonne journée,

**Comment by Pirhhot** [comment]
![image](https://github.com/user-attachments/assets/71864101-0250-4642-8791-e7119896998a)

**Reply by slecrom** [reply]
Merci @Pirhhot on verra lundi comment sélectionner les échantillons sur lesquels vous allez travailler.

**Comment by Nicolas-Doucet** [comment]
![image](https://github.com/user-attachments/assets/232a2295-f01b-4bae-b9af-5986ec17912b)

**Reply by slecrom** [reply]
Merci @Nicolas-Doucet nous verrons lundi pour mettre au propre les fichiers et les collections

**Comment by MichielTaw** [comment]
Bonjour, voici ma capture d'écran
![Capture d’écran 2024-11-29 162740](https://github.com/user-attachments/assets/f1e72909-7020-4b80-81a6-c4f159f02040)

**Reply by slecrom** [reply]
Merci @MichielTaw tout est bon pour lundi
